CONTRIBUTING.md

```diff
@@ -31,7 +31,7 @@ yarn install
 yarn bootstrap --core
 ```
 
-The bootstrap command will ask which sections of the codebase you want to bootstrap. Unless you're going to work with ReactNative or the Documentation, you can keep the default.
+The bootstrap command might ask which sections of the codebase you want to bootstrap. Unless you're going to work with ReactNative or the Documentation, you can keep the default.
```

Comment:
In fact, here it won't do it because the `--core` opttion is already specified. Just `yarn bootstrap` would ask those

Reply:
Well, removing "will" altogether seemed like too big of a change for me to make :)

Reply:
I mean, this section probably needs to be revisited. @ndelangen can you help?